client/components/BookCard.tsx

```diff
@@ -41,6 +42,7 @@ export default function BookCard(props: Props) {
         >
           <Rating size="small" name="read-only" value={3} readOnly /> 20 reviews
         </Typography>
+        <BookshelfActions bookData={book} iconsOnly={true} />
```

Comment:
```suggestion
        <BookshelfActions bookData={book} iconsOnly />
```

`={true}` could be emitted as per react community convention